js/src/rive.ts

```diff
@@ -366,6 +377,14 @@ class StateMachine {
     return names;
   }
 
+  /**
+   * Advances the state machine instance by a given time.
+   * @param time - the time to advance the animation by in seconds
+   */
+  public advance(time: number) {
+    this.instance.advance(time);
```

Comment:
FYI StateMachines will always apply while they advance (advance internally eventually calls apply, potentially multiple times). This is because they have to apply in order for to allow them to set values to use for the next iteration (until they're stable). Just in case you're expecting it not to...

Reply:
Ah cool cool, yeah I mainly added this because I noticed in the render loop we have some places where we do `animation.instance.blablabla` and same with state machine, so just abstracting the `.instance` part behind their respective classes so they all just do `animation.advance` or `stateMachine.advance`